config/settings.yml

```diff
@@ -5,6 +5,8 @@ timeouts:
   events:
     poll_for: 240
     poll_interval: 2
+  etd:
+    embargo_poll_interval: 1
```

Comment:
i was on the fence about using an existing timeout vs adding a new one.  this felt like the slightly safer bet, but happy to change.